.github/release_template.md

```diff
@@ -102,4 +102,9 @@ On Release Day (for final release):
 * [ ] Days later: Update the updater script ```clientupdater.php``` (check the crash reporter if auto update is a good idea or we need a new release)
 * [ ] Tell GCX to increment the minimum supported version for enterprise customers
 * [ ] Check if minimum.supported.desktop.version (https://github.com/owncloud/core/blob/master/config/config.sample.php#L1152) needs to be updated in server
+
+15 minutes after after release:
+* [ ] Test all advertised download links to have the expected version
+* [ ] Check for build errors in OBS
+* [ ] disable publishing in OBS
```

Comment:
should this be ```enable```?

Reply:
Yes, this is after we checked the packages for completeness and errors. Yes, we want to disable to prevent OBS automatic rebuild fuckup.